gotopkg.go

```diff
@@ -1,204 +0,0 @@
-package main
-
-import (
-	"flag"
-	"fmt"
-	"github.com/groob/plist"
-	"io"
-	"io/ioutil"
-	"log"
-	"os"
-	"os/exec"
-	"strings"
-	"syscall"
-	"time"
-)
-
-const RECIPE_FAILED_CODE = 70
-
-type AutopkgRunReport struct {
```

Comment:
In general I agree. If this weren't for one total plist (and only plist) parsing format (where I can't really fathom needing to seperately break things out further) then I'd of avoided inline structs. I like that the code is neater in the compact format, though that could be fixed by another package source file, I suppose.